jerry-core/ecma/base/ecma-helpers-string.c

```diff
@@ -1171,7 +1245,19 @@ ecma_string_get_char_at_pos (const ecma_string_t *string_p, /**< ecma-string */
   ecma_length_t string_length = ecma_string_get_length (string_p);
   JERRY_ASSERT (index < string_length);
 
-  lit_utf8_size_t buffer_size = ecma_string_get_size (string_p);
+  lit_utf8_size_t buffer_size;
+  bool is_ascii;
+  const lit_utf8_byte_t *chars_p = ecma_string_raw_chars (string_p, &buffer_size, &is_ascii);
+
+  if (chars_p != NULL)
```

Comment:
I found out. (ECMA_STRING_CONTAINER_UINT32_IN_DESC and ECMA_STRING_CONTAINER_HEAP_NUMBER has no raw string data)